src/pages/index.tsx

```diff
@@ -39,6 +41,21 @@ const HomePage: NextPageWithLayout = () => {
     }
   }, [router.query, signedIn, signedInOptimistic]);
 
+  useEffect(() => {
+    if(signedIn) {
+      isPassKeyAvailable().then((passKeyAvailable: boolean) => {
+        if(!passKeyAvailable) {
+          openToast({
+            title: 'Limited Functionality',
+            type: 'WARNING',
+            description: 'To access all account features, try using a browser that supports passkeys',
```

Comment:
Refer to the design, there is a link associated to the word `passkeys`. Do you want to confirm with designer?